### ISSUE TITLE
Grid View crashing while using Helium Theme 

### ISSUE THREAD

**Issue by X4Lo** [Lacro59/playnite-gameactivity-plugin#165]
I have a problem with GameActivity crashing every time I switch to grid view while using the theme **Helium**.

- Note 1: I'm using the plugin **darklinkpower's Grid View Converter**
- Note 2: I tried other themes Like **Mythic** & **Harmony** it works just fine (this means it only crashes while using **Helium**).
- Note 3: Using Playtine 10.1 / GameActivity 3.0 / Helium 1.32

I tried to change the theme > switch to grid view > close Playnite and switch back to **Helium** (so Playnite starts with grid view by default) and now it is crashing on the start.

Here is the last 60 line of playnite.log
```txt
23-09 21:25:16.697|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Image_Cache_Size_Saver\ImageCacheSizeSaver.psm1, version 2.12
23-09 21:25:16.723|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Links_Sorter\LinksSorter.psm1, version 2.6
23-09 21:25:16.740|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Extra_Metadata_tools_2e0349ed-6da2-4095-9457-4c9fb544551e\ExtraMetadataTools.psm1, version 2.11
23-09 21:25:16.760|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\NVIDIA_GE_GameStream_Export\NVIDIAGEGameStreamExport.psm1, version 2.12
23-09 21:25:16.802|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Game_Media_Tools\GameMediaTools.psm1, version 2.9
23-09 21:25:16.820|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Steam_Game_Importer\SteamGameImporter.psm1, version 2.9
23-09 21:25:16.856|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Steam_Trailers\SteamTrailers.psm1, version 2.14
23-09 21:25:16.874|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Twitch_Add_Link\TwitchAddLink.psm1, version 2.10
23-09 21:25:16.890|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\LibraryExporterPS_Builtin\LibraryExporter.psm1, version 2.0
23-09 21:25:17.858|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Log_Open_a755c30f-580a-491e-9584-6c5314f6e254\LogOpen.psm1, version 2.8
23-09 21:25:17.875|INFO |ExtensionFactory:Loaded script extension: O:\Tools\Playnite\Extensions\Game_Engine_Checker_7a21243e-c7cc-4ca7-85bd-f6f96f22e9db\GameEngineChecker.psm1, version 2.10
23-09 21:25:18.415|DEBUG|WindowFactory:Show window Playnite.DesktopApp.Windows.MainWindowFactory: 0ad14a5a-90bd-4bd1-ab14-070ebc6349b2
23-09 21:25:18.830|DEBUG|GamesCollectionViewEntry:Reloading collection item view properties.
23-09 21:25:18.899|INFO |GameDatabase:Opening db O:\Tools\Playnite\library
23-09 21:25:18.908|DEBUG|WindowFactory:Show dialog window Playnite.Windows.ProgressWindowFactory: 95550caf-7f84-4b1b-802e-e3ddf15ed9c0
23-09 21:25:19.282|WARN |ExecutionTimer:--- Timer 'DatabaseLoadCollections', 371 ms to complete.
23-09 21:25:19.798|INFO |DatabaseStats:Completely recalculating database statistics...
23-09 21:25:19.834|DEBUG|WindowFactory:Closing window Playnite.Windows.ProgressWindowFactory: 95550caf-7f84-4b1b-802e-e3ddf15ed9c0, True
23-09 21:25:21.532|ERROR|ExtensionFactory:Failed to execute OnApplicationStarted method from SuccessStory plugin.
System.NullReferenceException: Object reference not set to an instance of an object.
   at CommonPluginsShared.Models.TraceInfos..ctor(Exception ex) in H:\GitHub\Lacro59\playnite-successstory-plugin\source\playnite-plugincommon\CommonPluginsShared\Models\TraceInfos.cs:line 25
   at CommonPluginsShared.Common.LogError(Exception ex, Boolean IsIgnored, String Message, Boolean ShowNotification, String PluginName, String NotificationMessage) in H:\GitHub\Lacro59\playnite-successstory-plugin\source\playnite-plugincommon\CommonPluginsShared\Common.cs:line 219
   at SuccessStory.Clients.RetroAchievements.GetConsoleIDs() in H:\GitHub\Lacro59\playnite-successstory-plugin\source\Clients\RetroAchievements.cs:line 206
   at SuccessStory.SuccessStory.OnApplicationStarted(OnApplicationStartedEventArgs args) in H:\GitHub\Lacro59\playnite-successstory-plugin\source\SuccessStory.cs:line 1090
   at Playnite.Plugins.ExtensionFactory.NotifiyOnApplicationStarted() in C:\projects\playnite\source\Playnite\Plugins\ExtensionFactory.cs:line 727
23-09 21:25:21.607|DEBUG|BaseCollectionView:Updating collection view settings.
23-09 21:25:21.878|INFO |PlayniteApplication:Application 10.1.0.18590 started
23-09 21:25:24.746|ERROR|PlayniteApplication:Unhandled exception occured.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at GameActivity.Controls.PluginChartLog.<>c__DisplayClass59_1.<GetActivityForGamesLogGraphics>b__2() in H:\GitHub\Lacro59\playnite-gameactivity-plugin\source\Controls\PluginChartLog.xaml.cs:line 411
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at Playnite.DesktopApp.ProgramEntry.Main(String[] args) in C:\projects\playnite\source\Playnite.DesktopApp\ProgramEntry.cs:line 61
23-09 21:25:24.763|DEBUG|WindowFactory:Show dialog window Playnite.Windows.ExtensionCrashHandlerWindowFactory: db22e8fd-05a6-49ef-a449-4aca92ccb520
```

**Comment by BloodShed-Oni** [comment]
Similar problem here.

Playtine 10.2
GameActivity 3.0 
Helium 1.32
I do not use 'darklinkpower's Grid View Converter' however.

Here's my playnite.log
```
24-09 17:25:50.530|DEBUG|HttpFileCache:Returning https://steamcdn-a.akamaihd.net/steam/apps/227300/page.bg.jpg from file cache E:\Playnite474\cache\images\0eb891b26fb60e0c0841d607d042d516.jpg.
24-09 17:25:56.762|ERROR|PlayniteApplication:Unhandled exception occured.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at GameActivity.Controls.PluginChartLog.<>c__DisplayClass59_1.<GetActivityForGamesLogGraphics>b__2() in H:\GitHub\Lacro59\playnite-gameactivity-plugin\source\Controls\PluginChartLog.xaml.cs:line 411
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at Playnite.DesktopApp.ProgramEntry.Main(String[] args) in C:\projects\playnite\source\Playnite.DesktopApp\ProgramEntry.cs:line 61
24-09 17:25:56.782|DEBUG|WindowFactory:Show dialog window Playnite.Windows.ExtensionCrashHandlerWindowFactory: c38f9d3e-136e-43bc-9e8e-ed7f7c3159d5
```

**Comment by Lacro59** [comment]
fixed with #166
